requirements/base.in

```diff
@@ -0,0 +1,7 @@
+# TODO: these requirements should not be pinned so precisely.
```

Comment:
I know you only added this TODO for me, but now I am going to ask you to remove it. :)
In the other PR you mentioned, @cpennington had written:
> since edx-lint is intended to standardize pylint versions, that setup.py should pin particular pylint versions.
Rather than blocking this PR on the right wording for this comment, let's just remove the comment for now.